Modules/Scripted/SegmentStatistics/SegmentStatisticsPlugins/ScalarVolumeSegmentStatisticsPlugin.py

```diff
@@ -143,55 +144,55 @@ def getMeasurementInfo(self, key):
         # @moselhy also could not find DICOM quantity code for "median"
 
         info["voxel_count"] = \
-            self.createMeasurementInfo(name="Voxel count", description="Number of voxels", units="voxels",
-                                       quantityDicomCode=self.createCodedEntry("nvoxels", "99QIICR", "Number of voxels", True),
-                                       unitsDicomCode=self.createCodedEntry("voxels", "UCUM", "voxels", True))
+            self.createMeasurementInfo(name="Voxel count", description=_("Number of voxels"), units="voxels",
```

Comment:
Need to add a measurement Title property, which is displayed and translated.

Column header is a bit complicated. We need to introduce a "Title" property to tables (it can be stored in table column metadata). If column title is specified then on the GUI the column title is displayed instead of the column name. This requires changes in the Tables module. Once those changes are completed, we can complete translation of this module.